ui/src/components/drafts/form/themes/grommet/templates/ObjectFieldTemplate.js

```diff
@@ -19,11 +19,7 @@ let ObjectFieldTemplate = function(props) {
     return (
       <Box
         style={{
-          display: "grid",
-          justifyContent:
-            props.uiSchema["ui:options"] && props.uiSchema["ui:options"].align
-              ? props.uiSchema["ui:options"].align
-              : "center"
+          display: "grid"
```

Comment:
Sure we should remove?

Reply:
Probably not needed, thus why removed. I will restore them for this PR and check again in another since they are more concerned regarding the forms and not the workspaces. Both of them (below) were here due to playground changes